src/eos-autoupdater.c

```diff
@@ -212,7 +213,8 @@ update_stamp_file (guint update_interval_days,
     return;
   }
 
-  g_get_current_time (&mtime);
+  mtime.tv_sec = last_successful_update_secs;
```

Comment:
I’m not sure what you mean. In the `(randomized_delay_days == 0)` case, an update is scheduled immediately, and the `.stamp` file is not written until the update has completed successfully. What problem are you predicting here?